docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_privatelink/_index.md

```diff
@@ -0,0 +1,245 @@
+
+    - Azure VNET
+    - Azure Bastion
+    - Azure VPN Gateway and its public IP address
+    - Azure Linux Virtual Machine with a custom script extension that will run the [_install_arc_agent.sh_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script to Arc-enable the Azure VM.
+
+        > **Note: The [_install_arc_agent.sh_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/azure/linux/arm_template/scripts/install_arc_agent.sh) shell script will enable the OS firewall and set up new rules for incoming and outgoing connections. By default all incoming and outgoing traffic will be allowed, except blocking Azure IMDS outbound traffic to the *169.254.169.254* remote address.**
+
+5. User uses Azure Bastion to connect to Linux VM which will start the _install_arc_agent.sh_ script execution and will onboard the VM to Azure Arc.
+
+6. User tests private connectivity to Azure SQL server from the Azure Linux Virtual Machine.
+
+## Deployment
+
+As mentioned, this deployment will leverage ARM templates. You will deploy an ARM template per resource group that is responsible for creating all the Azure resources as well as the "on-premises" resources that will be onboarded to Azure Arc.
+
+- Before deploying the ARM template, login to Azure using AZ CLI with the ```az login``` command.
+
+- Create two resource groups in two different regions:
+
+  - Azure Resource Group
+  - On-premises Resource Group
+
+    ```shell
+    az group create --name <Name of the Azure resource group> --location <Azure region> --tags "Project=jumpstart_azure_arc_servers"
+    az group create --name <Name of the On-premise resource group> --location <Azure region> --tags "Project=jumpstart_azure_arc_servers"
+    ```
+
+    For example:
+
+    ```shell
+    az group create --name Arc-Azure-PL-demo --location "westeurope" --tags "Project=jumpstart_azure_arc_servers"
+    az group create --name Arc-Onpremise-PL-Demo --location "northeurope" --tags "Project=jumpstart_azure_arc_servers"
+    ```
+
+- The deployment will use two ARM template parameters file. Before initiating the deployment, edit the [*azuredeploy.parameters.json*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/privatelink/azuredeploy.parameters.json) file located in your local cloned repository folder and the edit the [*onpremisedeploy.parameters.json*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/privatelink/onpremisedeploy.parameters.json). Example parameters files are located [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/privatelink/azuredeploy.example.parameters.json) and [here](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/privatelink/onpremisedeploy.example.parameters.json).
```

Comment:
Can't we do this with a single ARM template?

Reply:
We can, but the template will be quite large and difficult to troubleshoot/iterate on in the future. Is there a reason linked templates should be avoided?

Reply:
To clarify, you only execute a single template here. The others are linked to the main template and deployed inline.

Reply:
Looking through the code, I believe this sections in the Readme needs to be updated to reflect the single parameters file used for the nested deployment.

Reply:
I didn't notice that so my bad. Although, The README is not doing a good job explaining it. 
Even though it's an implementation details, the README should make it clearer

Reply:
Agreed, code is good, README needs updating to reflect actual deployment requirements.

Reply:
I'll review and make some changes to make it clearer

Reply:
something weird happened and I had not commited my changes for the index, should be fixed now, sorry